envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoyControlHttp2Test.kt

```diff
@@ -0,0 +1,69 @@
+package pl.allegro.tech.servicemesh.envoycontrol
+
+import org.assertj.core.api.Assertions
```

Comment:
It should looks better when you change this import to: `import org.assertj.core.api.Assertions.assertThat`